lua/neoclip/sorted_set.lua

```diff
@@ -47,6 +47,21 @@ local insert = function(self, entry, opts)
     end
 end
 
+local replace = function(self, entry, newEntry)
```

Comment:
```suggestion
local replace = function(self, entry, new_entry)
```
just to keep the naming convention as lower_case

Reply:
Nice catch!